### PR TITLE
Resprites shotgun shell boxes (with fancy storage!)

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Resprites shotgun shell boxes
Increases the size of shell boxes to contain 8 shells

Also makes all shotgun shell boxes in the armory start out on shelves

## Why It's Good For The Game
Oh my god the old sprites were so big and clunky, they didn't fit on racks and were pretty garbage
These are fancy, show their contents (and can mix and match and all colors match up!)

## Images of changes
image

image

image

Slightly enlarged, you can see the mixing and matching
image

## Testing
See above
This took multiple dev team members almost 2 hours to get working. It works.

<hr>

### Declaration
image

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl: DGamerL, Contrabang
imageadd: Shotgun shell boxes have been resprited
tweak: Shotgun shell boxes can now hold 8 shells instead of 7
/:cl:

### CODE REVIEW

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -272,7 +272,7 @@
 				/obj/item/melee/baseball_bat/homerun = 50,
 				/obj/item/rcd/combat = 50,
 				/obj/item/weaponcrafting/gunkit/universal_gun_kit/sol_gov = 50,
-				/obj/item/storage/box/buck = 30, //Only seven shots, make them count
+				/obj/item/storage/fancy/shell/buck = 30, //Only seven shots, make them count
```
Comment (Contrabang) [inline code/game/objects/effects/spawners/lootdrop.dm:275]:
```suggestion
				/obj/item/storage/fancy/shell/buck = 30, //Only eight shots, make them count
```

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -699,76 +699,100 @@
...
+	. += "shell_box_front" // need to add another overlay to prevent from other overlays from showing on top
+
+/obj/item/storage/fancy/shell/tranquilizer
+	name = "ammunition box (Tranquilizer darts)"
+	desc = "A small box capable of holding seven shotgun shells."
+	icon_state = "tranqbox"
+
+/obj/item/storage/fancy/shell/tranquilizer/populate_contents()
+	for(var/i in 1 to 8)
+		new /obj/item/ammo_casing/shotgun/tranquilizer(src)
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/boxes.dm:733]: Make this a var on `/obj/item/storage/fancy/shell`, something like `shell_type`, null by default.
```
/obj/item/storage/fancy/shell/populate_contents()
	if(!shell_type)
		return
	for(var/i in 1 to storage_slots)
		new shell_type(src)
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Comment by 1080pCat** [comment]
> ## Changelog
> 
> 🆑 imageadd: shotgun shell boxes have been resprited /:cl:

makesure to add the shellcount change :)

**Comment by DGamerL** [comment]
> > ## Changelog
> > 🆑 imageadd: shotgun shell boxes have been resprited /:cl:
> 
> makesure to add the shellcount change :)

Done!

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 3]
```diff
@@ -699,77 +699,85 @@
 ////////////////
 /* Ammo Boxes */
 ////////////////
-/obj/item/storage/box/slug
-	name = "ammunition box (Slug)"
+
+// Empty type
+// All these can only hold their special ammo types and automatically are filled with them
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/boxes.dm:704]: need to update these 2 comments. Empty type is also a very vague comment and probably just should be removed

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 3 above)
Comment (DGamerL) [reply code/game/objects/items/weapons/storage/boxes.dm:704]: Removed!

**Review by Contrabang — APPROVED** [review]
LGTM. Update master when possible

**Comment by DGamerL** [comment]
Temporary closing because I'm over the limit (oops)

**Pull request merged**